modules/social_features/social_user_export/src/Plugin/UserExportPlugin/UserId.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Drupal\social_user_export\Plugin\UserExportPlugin;
+
+use Drupal\social_user_export\Plugin\UserExportPluginBase;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\user\UserInterface;
+
+/**
+ * Provides a 'UserId' user export row.
+ *
+ * @UserExportPlugin(
+ *  id = "user_id",
+ *  label = @Translation("User ID"),
+ *  weight = -500,
+ * )
+ */
+class UserId extends UserExportPluginBase {
+
+  use StringTranslationTrait;
+
+  /**
+   * Returns the header.
+   *
+   * @return \Drupal\Core\StringTranslation\TranslatableMarkup|string
+   *   The header.
+   */
+  public function getHeader() {
+    return $this->t('User Id');
```

Comment:
Nitpicking, but shall we make it `User ID`?

Reply:
sure